guillermo_martinez_folder/restaurants_challenge.py

```diff
@@ -0,0 +1,107 @@
+print("Challenge: Favourite Restaurants")
+
+# print()
+
+print("Question 1")
+
+print("Below is a dictionary consisting of details of 1 restaurant fetched from Yelp. Go through the dictionary and print out the following 3 pieces of information about the restaurant: \n1. The latitude and longitude of Four Barrel Coffee \n2. The complete address of Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code. \n3. The website of Four Barrel Coffee.")
+
+restaurant = {
+    "name": "Four Barrel Coffee",
+    "url": "https://www.yelp.com/biz/four-barrel-coffee-san-francisco",
+    "latitude": 37.7670169511878,
+    "longitude": -122.42184275,
+    "city": "San Francisco",
+    "country": "US",
+    "address2": "",
+    "address3": "",
+    "state": "CA",
+    "address1": "375 Valencia St",
+    "zip_code": "94103",
+    "distance": 1604.23,
+    "transactions": ["pickup", "delivery"]
+}
+# TODO: Write code to print the latitude and longitude of Four Barrel Coffee.
+print(f'The {restaurant["name"]} is located at {restaurant["latitude"]} and {restaurant["longitude"]}') 
+
+# TODO: Write code to print the complete address of the Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+print(f'The {restaurant["name"]} is located at {restaurant["address1"]}, {restaurant["city"]}, {restaurant["state"]}, and {restaurant["zip_code"]}')
+
+print()
+print()
+# TODO: Write code to print the URL of the website of Four Barrel Coffee.
+print(f'The website is {restaurant["url"]}')
+
+
+print()
+
+print("Question 2")
+
+# TODO: Create a new empty dictionary called fav_restaurants.
+fav_restaurants = {}
+#fav_restaurants2 = {}
+#fav_restaurants3 = {}
+
+# TODO: Choose 3 of your most favourite restaurants in NYC and add the following information for those restaurants inside fav_restaurants:
+#         1. Name of the restaurant - Should be the key of the dictionary
+#         2. Address of the restaurant - 1st value of the list
+#         3. Favourite dish in that restaurant - 2nd value of the list
+#         4. Opening and closing hours of that restaurant - 3rd value of the list
+fav_restaurants['Chicken_spot'] = "180 Brooklyn Ave, Brooklyn, NY 11221: centerbreasts: 24 hours"
+print(fav_restaurants)
+print()
+fav_restaurants['Pizzeria'] = "180 Queens Ave, Queens, NY 10201: Reg slice: 10am - 9pm"
+print()
+print(fav_restaurants)
+print()
+fav_restaurants['Halal'] = "180 Bronx Ave, Bronx, NY 10056: Gyro: 12pm - 10pm"
```

Comment:
it may have made it easier for question 4 if these values were lists, but your solution still works so no biggie!